blackbox/docs/sql/reference/select.txt

```diff
@@ -31,7 +31,9 @@ SELECT retrieves rows from a table. The general processing of SELECT
 is as follows:
 
 - The FROM item points to the table where the data should be retrieved
-  from.
+  from. If no FROM item is specified, the query is instead executed against
+  ``sys.cluster``. This is so that queries that are commonly used in connection
```

Comment:
imho i would not add use-case docs in the reference. 
however i would also add a pointer where sys.cluster is documented and also link from here to the sys.cluster documentation